translations/monero-core_pl.ts

```diff
@@ -459,12 +459,12 @@
     <message>
         <location filename="../LeftPanel.qml" line="439"/>
         <source>Prove/check</source>
-        <translation type="unfinished"></translation>
+        <translation>Sprawdź płatność</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="440"/>
         <source>K</source>
-        <translation>P</translation>
+        <translation>D</translation>
```

Comment:
I know that the letters are symbols for the tabs in the left menu and that when you press Ctrl, you can switch to a tab with corresponding letter. I translated the letters based on tabs translations. So H is still H for History tab because it's also Historia in Polish. But Address Book (B) is Książka adresowa, so I translated it to K. It would be weird for Polish users to have to click B for Książka adresowa. I agreed on that solution with @erciccione but if you two decide that they shouldn't be translated, then I'll change that without a word. (but if so then why even make these letters translatable if you shouldn't translate them)

Reply:
I think it's always better to translate even shortcuts if possible. This is a very minor matter, but if we don't care to localize shortcuts (i honeslty don't) we should just unflag them so they don't get translated @jonathancross , but in this case, imo, the translation is correct

Reply:
If we begin "translating" the letters here, users will have no idea what the correct keyboard shortcut is.  That would defeat the purpose of these characters being shown at all.  It seems users will just need to memorize the (somewhat strange[1]) key mapping.

[1] The keyboard shortcuts don't make a whole lot of sense in English either, but at least the letter shown opens that menu item:
`B` for Address Book
`D` for Advanced
`K` for Prove/check
`I` for Sign/Verify
`E` for Settings

In the future, we might remap to numbers: Ctrl-1, Ctrl-2, etc. to avoid these problems.

Reply:
Users will probably choose 1 lang version and not switch between versions. So Polish users can memorize Polish shortcuts just like English users can memorize English ones.
And in general memorazing is not necessary as the symbols are shown after pressing Ctrl. In Polish it would be (the same order):
`K` for Książka adresowa
`Z` for Zaawansowane
`D` for Sprawdź płatność (because of Prove meaning Dowiedź; this one can be changed if somebody has a better idea but S and P are already taken)
`P` for Podpisz/weryfikuj
`U` for Ustawienia
`W` for Wyślij (Send)
`O` for Otrzymaj (Receive)
`S` for Seed i klucze (Seed & keys)
If we not translate the letters then users might actually have to memorize them.

Reply:
Maybe I misunderstood your answer to my first question.
So you are saying that updating the translation file: `U` _for Ustawienia_ will then allow you to use `Control-U` to select the "Settings" tab? If so, then this is fine.  I just didn't want it to show `U` when the user must type Control `E`.

Reply:
>In the future, we might remap to numbers: Ctrl-1, Ctrl-2, etc. to avoid these problems.

I agree, that would be the best solution

Reply:
Now it all makes sense, Jonathan. :D I'll try to build monero-gui with my translations and see if the shortcuts work with the translated symbols. But you're probably right and people would still have to use English shortcuts for other translations. So a very good call! Wouldn't think about it myself. I'll make one PR when I make all the changes that were suggested.

Reply:
Looking at the code briefly, I think it might actually "just work" - but we should confirm.

Reply:
@einsteinsfool you can edit directly on this PR, than we can test it, no needs of another PR

Reply:
@jonathancross You were correct the first time. The letters shouldn't be translated because users still have to use English letter shortcuts. I managed to finally build monero-gui on Ubuntu 16.04 VM (with changes from this PR) and I confirm that. So it would probably be a good idea to suggest other translators in other PRs to make sure the letters in their language match the English letters.
@erciccione Sure. I'll update this PR tomorrow with the changes suggested here, changing "proces" to "demon" in all phrases and fixing 1 spelling error (Śieć -> Sieć).

Reply:
Okay, thanks for confirming my suspicion.